src/Azure.Functions.Cli/Azure.Functions.Cli.csproj

```diff
@@ -71,10 +71,10 @@
     <PackageReference Include="KubeClient" Version="2.1.5" />
     <PackageReference Include="KubeClient.Extensions.KubeConfig" Version="2.1.5" />
     <PackageReference Include="Microsoft.AspNetCore.DataProtection" Version="2.1.1" />
-    <PackageReference Include="Microsoft.Azure.Functions.JavaWorker" Version="1.1.1-beta03-10111" />
-    <PackageReference Include="Microsoft.Azure.Functions.NodeJsWorker" Version="1.0.0-beta4-10082" />
+    <PackageReference Include="Microsoft.Azure.Functions.JavaWorker" Version="1.2.0-beta01-SNAPSHOT" />
```

Comment:
sorry, is it supposed to stay "SNAPSHOT" in the version?

Reply:
I believe that's right -- @pragnagopa?

Reply:
okay, just making sure :)